Calibration/EcalCalibAlgos/src/ECALpedestalPCLHarvester.cc

```diff
@@ -50,8 +51,10 @@ void ECALpedestalPCLHarvester::dqmEndJob(DQMStore::IBooker& ibooker_, DQMStore::
         ped.mean_x12=mean;
         ped.rms_x12=rms;
 
-        // if bad channel or low stat skip
-        if(ch->getEntries()< minEntries_ || !checkStatusCode(id)){
+        float diff = abs(mean-oldped.mean_x12);
```

Comment:
hi @argiro - please use std::abs (presumably you don't want integer abs here in any case.